tensorflow/training/docker/2.3.0/py3/cu102/Dockerfile.gpu

```diff
@@ -178,6 +179,12 @@ RUN ln -s /etc/ssl/certs/*.* /usr/local/ssl/certs/
 RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
  && ln -s $(which ${PIP}) /usr/bin/pip
 
+# Validate tf binaries hash
+RUN wget ${TF_URL}
+RUN ${PIP} hash tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl | tee hashresult.txt
+RUN rm tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl
+RUN HASH=$(grep -oE 'sha256[^ ]*' hashresult.txt) && if test "$TF_HASH" = "$HASH"; then echo Hash Verified; else echo $HASH ---- $TF_HASH Hash Validation Failed && exit 1; fi
```

Comment:
Please merge all these `RUN` statements into a single RUN statement joined by `&&`. This will reduce the net image size.